Gemfile

```diff
@@ -99,6 +99,8 @@ gem 'riiif', '~> 1.1'
 
 gem 'iiif_manifest', '~> 0.1.2'
 
+gem 'carrierwave-aws'
```

Comment:
Should this go in the `:aws` bundler group so that folks who use Hyku locally (e.g., via Docker) don't pick up this dependency?